kudo-spark-operator/operator/operator.yaml

```diff
@@ -77,12 +114,22 @@ tasks:
   spec:
     resources:
     - spark-operator-deployment.yaml
+- kind: Delete
+  name: spark-cleanup
+  spec:
+    resources:
+      - spark-operator-deployment.yaml
```

Comment:
not sure if it's really necessary to delete operator deployment as a part of the upgrade - I believe KUDO should be able to update the existing deployment and then k8s will do the rest

Reply:
thanks, removed that